src/execution/Execution.ts

```diff
@@ -2,7 +2,6 @@ import { CalculationOptions, RawExecutionData, DebugOutput } from '../types/Calc
 
 // import { PatientSource } from 'cql-exec-fhir';
```

Comment:
```suggestion
```
not your addition, but looks like this can be removed

Reply:
Absolutely can be removed here since this file only speaks generic PatientSource.